docs/advanced_topics/performance.rst

```diff
@@ -59,13 +59,12 @@ The overhead from reading and compiling templates can add up. In some cases a si
                 ('django.template.loaders.cached.Loader', [
                     'django.template.loaders.filesystem.Loader',
                     'django.template.loaders.app_directories.Loader',
+                    'path.to.custom.Loader',
                 ]),
             ],
         },
     }]
 
-There is a caveat associated with this loader though. Changes to a template file will not be picked up once it is cached. This means that this loader should *not* be enabled during development.
```

Comment:
I think it is worth leaving this note since specifically wrapping the loader with the cached loader will result in the caveat it describes

Reply:
i feel it's a bit of a tautology as it is explained well in the linked django docs, but i have no strong preference, dont mind if it stays.

Reply:
It is, however you would be surprised how many people will overlook that!